compiler/z/codegen/S390Peephole.cpp

```diff
@@ -1525,18 +1525,10 @@ bool swapOperands(TR::Register * trueReg, TR::Register * compReg, TR::Instructio
       }
       return true;
    }
-
-void insertLoad(TR::Compilation * comp, TR::CodeGenerator * cg, TR::Instruction * i, TR::Register * r)
+void insertLoad(TR::CodeGenerator * cg, TR::Instruction * i, TR::Register * r)
    {
-   switch(r->getKind())
-     {
-     case TR_FPR:
-       new (comp->trHeapMemory()) TR::S390RRInstruction(TR::InstOpCode::LDR, i->getNode(), r, r, i, cg);
-       break;
-     default:
-       new (comp->trHeapMemory()) TR::S390RRInstruction(TR::InstOpCode::LR, i->getNode(), r, r, i, cg);
-       break;
-     }
+   TR::Compilation * comp = cg->comp();
```

Comment:
Now that the allocation is in `generateRRInstruction`, this variable is unused and can be removed.

Reply:
sorry I applied your previous suggestion without noticing this, thanks for pointing this out it's been resolved now